tests/FakeGRPC.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14-slim
+
+WORKDIR /app
+
+COPY .npmrc .
```

Comment:
generally, this package considered open-source. so we need to consider if we want to do it.